FreeCRMTest/src/main/java/com/crm/qa/pages/RushiPage.java

```diff
@@ -0,0 +1,9 @@
+package com.crm.qa.pages;
+
+import com.crm.qa.base.TestBase;
+
+public class RushiPage extends TestBase {
```

Comment:
Please add all the actions of rushi page.

Reply:
Done.Added the logic